packs/equipment/shield-of-the-unified-legion.json

```
@@ -49,22 +49,41 @@
             }
         },
         "propertyRune1": {
-            "value": ""
+            "value": null
         },
         "propertyRune2": {
-            "value": ""
+            "value": null
         },
         "propertyRune3": {
-            "value": ""
+            "value": null
         },
         "propertyRune4": {
-            "value": ""
+            "value": null
         },
         "quantity": 1,
         "resiliencyRune": {
-            "value": ""
-        },
-        "rules": [],
+            "value": null
+        },
+        "rules": [
+            {
+                "effects": [
+                    {
+                        "affects": "allies",
+                        "events": [
+                            "enter"
+                        ],
+                        "includesSelf": false,
+                        "uuid": "Compendium.pf2e.equipment-effects.ah41XCrV4LFsVyzl"
```

```suggestion
                        "uuid": "Compendium.pf2e.equipment-effects.Item.Effect: Shield of the Unified Legion"
```
This should have been fixed by extracting.

---

It wasn't recognized as UUID because it's missing the `.Item` component.